### ISSUE TITLE
Ignore access=private for all quests

### ISSUE THREAD

**Issue by Atrate** [Helium314/StreetComplete#607]
**Use case**

I don't really get why the original SC ignores quests for things tagged access=private. In most cases it's e.g. a service road I can already see so I can map the surface and other quests without a problem and without tresspassing.

**Proposed Solution**

While SCEE allows to edit the overpass query for individual quests, with 216 quest types and counting it has become unfeasible to remove the access=private filters on all of them. It'd be nice if there was a toggle in the settings that did it for all of them simultaneously.

**Comment by Helium314** [comment]
It's also a lot of work for me (though it seems that less than 100 quests have a _private_ filter), and with questionable benefit.
I'm not going do implement this, but I'm willing to accept a PR under the condition that it minimizes the chances of merge conflicts when SC filters are changed.
(I'm also not going to update it when new quests are added to SC)

**Comment by mnalis** [comment]
Yes, it would seem like too much work and maintenance to me too (unless it was some smart-regexp option which searched for `and access ~ no|private` and similar strings in element selector strings and replaced them with empty strings (but then again such generic approach might also backfire).

What I'm more worried about is that removing private restrictions should **not** IMHO apply to all quests. Sure, I can often see `surface`, `sidewalk` and `cycleway` on that `access=private` road, so removing private restrictions there might be useful in some cases... 
But `smoothness` is harder (well, at least for my eyesight) - especially if that road segment is not rather short, I cannot reliably tell exact surface quality on its **whole** length (without walking it down) 

Even more problematic are quests like `drinking_water=*` on that `access=private` well? Or `internet_access=*` in that `access=private` chalet? Or `payment:*` on that `access=private` club? I don't think can answer those (without getting in problems).

**Comment by Helium314** [comment]
> I don't think can answer those (without getting in problems).

Though maybe you know the right people and can thus legally visit some of those locations...

But actually I think everything would be fine, as you have to explicitly enable the setting. So you should know that quests can be asked for places you're not allowed to access.

**Comment by RubenKelevra** [comment]
That's actually a good idea. But maybe we should ask in the SC project itself before we do something like this, with the same argument?

I modified a lot of quests to do exactly this in SCEE, because I most of the time can add information like surface to those roads as well.

**Comment by Atrate** [comment]
> That's actually a good idea. But maybe we should ask in the SC project itself before we do something like this, with the same argument?

Feel free to do that, but I am quite convinced that this would never fly with the original SC.

**Comment by rusty-snake** [comment]
> because I most of the time can add information like surface to those roads as well.

Well, surface of `access=private` roads can be edited with SC in the surface-overlay. Which is also a pretty good solution, no dangling unanswerable quests, but if you can answer you can just edit it.
